src/environment/isolator.ts

```diff
@@ -83,6 +98,8 @@ export default class Isolator {
       });
     }
     const writeToPath = opts.writeToPath;
+    const installNpmPackages = typeof opts.installNpmPackages === 'undefined' ? true : opts.installNpmPackages;
+    // default should be true
```

Comment:
please move the comment to be above the code line, it's above the code across the whole repo. (relevant for other comments as well)